https-bastion.md

```diff
@@ -0,0 +1,138 @@
+# HTTPS Bastion
+
+This document specifies an HTTPS bastion, a service that acts as a reverse proxy
+accepting connections from both clients and backends and routing requests from
+the former to the latter.
```

Comment:
Suggestion: break up into >1 sentence, a bit heavy as is.

Reply:
Toyed with this a bit and failed to make it both lighter and not stuttering.